models/sdss_binary_results

```diff
@@ -0,0 +1,3 @@
+sdss_binary
+epoch=229-val_loss=5.414.ckpt
```

Comment:
@ismael-mendoza After retraining binary with a VAE+flow, the loss is around where it was before. So I think it was a red flag that the loss went down a lot.